src/accelerate/components/List.tsx

```diff
@@ -129,27 +186,38 @@ export default function List( props: Props ) {
 							}
 
 							return (
-								<tr key={ post.id }>
-									<td className="record-traffic">
-										<div className="traffic-bar" style={ { right: `${ 100 - ( post.views / maxViews * 100 ) }%` } }></div>{ compactMetric( post.views ) }
+								<tr key={post.id}>
+									<td className='record-thumbnail'>
+										{ post.thumbnail && (
+											<img src={ post.thumbnail } alt={ post.title } width="100" height="50" />
```

Comment:
This width & height are different to what was in the CSS but should be removed as we need to support various aspect ratios. Use a wrapper div that has a specific width & height then `object-fit` will work properly.

Reply:
`object-fit` only applies to replaced elements like `<img/>` rather than their containers, and it assumes the element has an explicit width/height so the fitting would work. Tried a couple of scenarios and this was the best fit that worked as expected. Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit